### DISCUSSION TITLE
FireBase version 9 configuration

### DISCUSSION THREAD

**Discussion by justbazz4** [firebase/firebase-admin-node#1923] [General]
Below is my setup in  firebase.js for a register and signIn page, my app is rendering a WEBPACK error :  

import { initializeApp } from 'firebase/app'
import firebase from './firebase';
import "firebase/firestore";

const firebaseConfig = {
    apiKey: "REDACTED",
    authDomain: "REDACTED.firebaseapp.com",
    projectId: "REDACTED",
    storageBucket: "REDACTED.appspot.com",
    messagingSenderId: "REDACTED",
    appId: "REDACTED"
  };

  const firebaseApp = firebase.initializeApp(firebaseConfig);
  const db = firebaseApp.firestore();
  const auth = firebase.auth();

  export { auth, initializeApp };
  export default db;

**Comment by justbazz4** [comment]
Below is my Signup page . What am i doing wrong on this code . This is my first time using firebase . 



import React, { useRef } from 'react';
import { auth } from "../firebase";
import './SignUpScreen.css';


function SignUpScreen() {
  const emailRef = useRef(null);
  const passwordRef = useRef(null);

    const register =(e) => {
        e.preventDefault();

        auth.createUserWithEmailAndPassword(
            emailRef.current.value,
            passwordRef.current.value
        ).then((authUser)=>{
              console.log(authUser)
        }).catch((error)=>{
           alert(error.message);
        });
    };

    const signIn = (e) => {
        e.preventDefault();
    }

  return (
    <div className='signupScreen'>
        <form>
            <h1>Sign In</h1>
            <input ref={emailRef} placeholder='Email' type='email' />
            <input ref={passwordRef} placeholder='Password' type='password' />
            <button type='sumbit' onClick={signIn}>Sign In</button>

            <h4>
                <span className='signupScreen__gray'>New to Netflix? </span>
                 <span className='signupScreen__link' onClick={register}>Sign Up now.</span>
             </h4>
        </form>
  </div>
  )
}

export default SignUpScreen

**Comment by justbazz4** [comment]
This is my code below 

`import React, { useRef } from 'react';
import { auth } from "../firebase";
import './SignUpScreen.css';


function SignUpScreen() {
  const emailRef = useRef(null);
  const passwordRef = useRef(null);

    const register =(e) => {
        e.preventDefault();

        auth.createUserWithEmailAndPassword(
            emailRef.current.value,
            passwordRef.current.value
        ).then((authUser)=>{
              console.log(authUser)
        }).catch((error)=>{
           alert(error.message);
        });
    };

    const signIn = (e) => {
        e.preventDefault();
    }

  return (
    <div className='signupScreen'>
        <form>
            <h1>Sign In</h1>
            <input ref={emailRef} placeholder='Email' type='email' />
            <input ref={passwordRef} placeholder='Password' type='password' />
            <button type='sumbit' onClick={signIn}>Sign In</button>

            <h4>
                <span className='signupScreen__gray'>New to Netflix? </span>
                 <span className='signupScreen__link' onClick={register}>Sign Up now.</span>
             </h4>
        </form>
  </div>
  )
}

export default SignUpScreen`

**Comment by lahirumaramba** [comment]
Hi, this is the repo for Firebase Admin Node.js SDK. It looks like you are using the Firebase JS client SDK 
Please ask JS SDK related questions [firebase/firebase-js-sdk](https://github.com/firebase/firebase-js-sdk) or use Stack Overflow for usage questions.

Also, please don't share any sensitive project credentials (APIKey, App ID etc.) on public posts. I have redacted that information from your original post.
